src/main/java/de/tum/in/www1/artemis/service/connectors/BambooService.java

```diff
@@ -499,6 +504,11 @@ public ConnectorHealth health() {
         return health;
     }
 
+    @Override
+    public Optional<String> getWebhookUrl(String projectKey, String buildPlanId) {
+        return Optional.empty();
```

Comment:
add comment that this is not needed, explain in detail (potentially with link to Bamboo documentation) that Bitbucket and Bamboo communicate on their own

Reply:
I added a comment and a link